src/App.scss

```
@@ -1 +1,5 @@
-// styles go here
+.App h1 {
```

```suggestion
h1 {
```
А вообще лучше добавить класс, и юзать его для стилизации

---

css is a pain